frontend/pages/admin/AppSettingsPage/AppSettingsPage.tsx

```diff
@@ -104,8 +90,7 @@ const AppSettingsPage = (): JSX.Element => {
   return (
     <div className={`${baseClass} body-wrap`}>
       <p className={`${baseClass}__page-description`}>
-        Set your organization information, Configure SAML and SMTP, and view
-        host enroll secrets.
```

Comment:
@noahtalerman I made these changes to the header description of the app settings page because we are no longer displaying the host enroll secrets here. Let me know if there's better wording you would like to use.

Reply:
>  we are no longer displaying the host enroll secrets here

@RachelElysia thank you for catching this! 

The changes you made look good to me (bonus points for sentence case fixes 🎸).

Thank you for looping me in :)